stripe/res/values/styles.xml

```diff
@@ -62,4 +62,16 @@
 
     <style name="Stripe.BecsDebitWidget.MandateAcceptanceTextView"
         parent="Stripe.Base.BecsDebitWidget.MandateAcceptanceTextView" />
+
+    <style name="StripeBaseTheme.Transparent" parent="@style/Theme.MaterialComponents.DayNight.NoActionBar">
```

Comment:
should this be in `themes.xml`?

Reply:
we don't actually have a `themes.xml` but happy to take this opportunity to create one if you like

Reply:
You can do it in a separate PR if you'd like

Reply:
yeah, sounds good